functions/go/generate-services/service-map.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: blueprints.cloud.google.com/v1alpha1
```

Comment:
Do we have source of truth for this file somewhere? How can we ensure it's up-to-date?